app/models/ems_refresh/save_inventory_networks.rb

```diff
@@ -0,0 +1,195 @@
+#
```

Comment:
@aufi seems like this whole file is duplicated? the right one is app/models/ems_refresh/save_inventory_network.rb